### PR TITLE
Update security page

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

This PR updates our security page. The intention is to:

- Detail how we handle security issues.
- Detail on where to report them.
- Provide information on non-qualifying issues
- Information on past published security advisories.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/security/index.markdown [hunk 1]
```diff
@@ -1,11 +1,81 @@
 ---
 title: "Security"
-description: "Information about disclosing security vulnerabilities in Home Assistant."
+description: "Home Assistant takes its security seriously. This page contains information about how we handle security issues, how to report them, and information on past security issues."
```
Comment (reedy) [inline source/security/index.markdown]: "and also information"?

File: source/security/index.markdown [hunk 2]
```diff
@@ -1,11 +1,81 @@
...
 ---
 
-If you think that you have found a security vulnerability in Home Assistant, please disclose it to us via our security e-mail address at [security@home-assistant.io](mailto:security@home-assistant.io).
+Home Assistant takes its security seriously. We will do everything in our power to ensure that our users are safe.
 
-Please do not make vulnerabilities public without notifying us and giving us at least 3 days to respond.
+This page is intended to provide information about how we handle security issues and information about security issues we have [handled in the past](#past-advisories).
```
Comment (reedy) [inline source/security/index.markdown]: "handle issues that are reported to us" or "handle issues whenthey are reported to us"; can't handle those we don't know about

File: source/security/index.markdown [hunk 3]
```diff
@@ -1,11 +1,81 @@
...
 
-If you are going to write about Home Assistant's security, please get in touch, so we can make sure that all claims are correct.
-  
+## Reporting a vulnerability
+
+So, you have found a security vulnerability in Home Assistant? Please, be sure to **responsibly disclose** it to us by [reporting a vulnerability using GitHub's Security Advisory](https://github.com/home-assistant/core/security/advisories/new).
```
Comment (reedy) [inline source/security/index.markdown]: Presuming it's fine for reports about any HA component to be reported in the core repo then?

https://en.wikipedia.org/wiki/Coordinated_vulnerability_disclosure might be a good link for "responsibly disclose"

File: source/security/index.markdown [hunk 4]
```diff
@@ -1,11 +1,81 @@
...
+
+**DO NOT MAKE A PUBLIC ISSUES FOR SECURITY VULNERABILITIES!**
+
+We are mostly interested in reports by actual Home Assistant users that are familiar with the platform, but all high quality contributions are welcome. Please do your best to describe a clear and realistic impact for your report.
+
+For the sake of the security of our users, please 🙏 do not make vulnerabilities public without notifying us and giving us at least 90 days to release a fixed version. We will do our best to respond to your report within 7 days and also to keep you informed of the progress of our efforts to resolve the issue, but understand that Home Assistant, like many open source projects, is relying heavily on volunteers that aren't full-time resources. We may not be able to respond as quickly as you would like due to other responsibilities.
+
+If you are going to write about Home Assistant’s security, please get in touch, so we can ensure that all claims are correct.
```
Comment (reedy) [inline source/security/index.markdown]: How do you want people to get in touch?

File: source/security/index.markdown [hunk 5]
```diff
@@ -1,11 +1,81 @@
...
+
+### Non-qualifying vulnerabilities
+
+We will not accept reports of vulnerabilities of the following types:
+
+- Reports from automated tools or scanners.
```
Comment (reedy) [inline source/security/index.markdown:26]: Presumably the reports are ok, if substantiated/tested/verified etc? Just not the reports that are "here's the output of some tool I don't understand, and haven't checked anything. It says you have issues"?

File: source/security/index.markdown [hunk 6]
```diff
@@ -1,11 +1,81 @@
...
+- Theoretical attacks without proof of exploitability.
+- Attacks that are the result of a third-party application or library (these should instead be reported to the library maintainers).
```
Comment (reedy) [inline source/security/index.markdown:28]: What if HA is including an older/vulnerable version of the library?

File: source/security/index.markdown [hunk 7]
```diff
@@ -1,11 +1,81 @@
...
+- Social engineering.
+- Attacks that require the user to have access to the Home Assistant host system.
```
Comment (reedy) [inline source/security/index.markdown:30]: What does "access" mean in this case? Physical? SSH? Valid login details (with permission)?

Especially wrt the later points about "logged in users" as below.

File: source/security/index.markdown [hunk 8]
```diff
@@ -1,11 +1,81 @@
...
+- Attacks involving physical access to a user’s device, or involving a device or network that’s already seriously compromised (like, man-in-the-middle).
+- Attacks that require the user to install a malicious other software, like a third-party integration, add-on, or plugin.
+- Attacks that the user can only perform on themselves.
+- Privilege escalation attacks for logged in users. Home Assistant assumes every user is trusted and does not enforce user privileges. It assumes every logged in user has the same access as an owner account ([more information](/docs/authentication/#user-accounts)).
+
+### Supported versions
+
+We only accept reports against the latest stable & official versions of Home Assistant or any versions beyond that currently in development or beta test. The latest version can be found [here](https://github.com/home-assistant/core/releases).
```
Comment (reedy) [inline source/security/index.markdown]: "that are currently in development"

`The latest stable version can be found on [GitHub](https://github.com/home-assistant/core/releases).`

(as below about `[here]` links)

File: source/security/index.markdown [hunk 9]
```diff
@@ -1,11 +1,81 @@
...
+
+We do not accept reports against forks of Home Assistant.
+
+### Severity scoring
+
+If you are familiar with [CVSS3.1](https://www.first.org/cvss/v3.1/specification-document), please provide the vulnerability score in your report in the shape of a vector string. There’s a calculator [here](https://www.first.org/cvss/calculator/3.1). If you are unsure how or unable to score a vulnerability, state that in your report, and we will look into it.
```
Comment (reedy) [inline source/security/index.markdown]: It's generaly advised against using "here" or "click here" for links. `Use a [CVSS calculator](https://www.first.org/cvss/calculator/3.1) to help.` or something would be better

File: source/security/index.markdown [hunk 10]
```diff
@@ -1,11 +1,81 @@
...
+
+If you intend to provide a score, please familiarize yourself with CVSS first (we strongly recommend reading [Specification](https://www.first.org/cvss/v3.1/specification-document) and [Scoring Guide](https://www.first.org/cvss/v3.1/user-guide#Scoring-Guide)), as we will not accept reports that use it incorrectly.
+
+### Public disclosure & CVE assignment
+
+We will publish GitHub Security Advisories and through those, will also request CVEs, for valid vulnerabilities that meet the following criteria:
+
+- The vulnerability is in Home Assistant itself, not a third-party library.
```
Comment (reedy) [inline source/security/index.markdown:52]: wrt the above points...

What counts as a third party library in this case? How would a reporter know/be able to deduce?

What would be expected if the report was about HA using a vulnerable third party library, and whatever the issue was being exploited was exposed through HA...

HA would update, but I'd suspect a CVE would probably not be needed, but a GHSA might.

File: source/security/index.markdown [hunk 11]
```diff
@@ -1,11 +1,81 @@
...
+
+### Severity scoring
+
+If you are familiar with [CVSS3.1](https://www.first.org/cvss/v3.1/specification-document), please provide the vulnerability score in your report in the shape of a vector string. There’s a calculator [here](https://www.first.org/cvss/calculator/3.1). If you are unsure how or unable to score a vulnerability, state that in your report, and we will look into it.
+
+If you intend to provide a score, please familiarize yourself with CVSS first (we strongly recommend reading [Specification](https://www.first.org/cvss/v3.1/specification-document) and [Scoring Guide](https://www.first.org/cvss/v3.1/user-guide#Scoring-Guide)), as we will not accept reports that use it incorrectly.
+
+### Public disclosure & CVE assignment
+
+We will publish GitHub Security Advisories and through those, will also request CVEs, for valid vulnerabilities that meet the following criteria:
```
Comment (reedy) [inline source/security/index.markdown:50]: Not sure https://docs.github.com/en/code-security/security-advisories/working-with-repository-security-advisories/about-repository-security-advisories is hte best link, but some cross reference to the GHSA procedure etc would be good (incase the reporter isn't actually familar with them/the process)

File: source/security/index.markdown [hunk 12]
```diff
@@ -1,11 +1,81 @@
...
+
+We only accept reports against the latest stable & official versions of Home Assistant or any versions beyond that currently in development or beta test. The latest version can be found [here](https://github.com/home-assistant/core/releases).
+
+We do not accept reports against forks of Home Assistant.
+
+### Severity scoring
+
+If you are familiar with [CVSS3.1](https://www.first.org/cvss/v3.1/specification-document), please provide the vulnerability score in your report in the shape of a vector string. There’s a calculator [here](https://www.first.org/cvss/calculator/3.1). If you are unsure how or unable to score a vulnerability, state that in your report, and we will look into it.
+
+If you intend to provide a score, please familiarize yourself with CVSS first (we strongly recommend reading [Specification](https://www.first.org/cvss/v3.1/specification-document) and [Scoring Guide](https://www.first.org/cvss/v3.1/user-guide#Scoring-Guide)), as we will not accept reports that use it incorrectly.
```
Comment (reedy) [inline source/security/index.markdown]: "reading the [Specification]"

File: source/security/index.markdown (hunk 5 above)
Comment (frenck) [reply source/security/index.markdown:26]: As it says... exactly that.

File: source/security/index.markdown (hunk 6 above)
Comment (frenck) [reply source/security/index.markdown:28]: Then we need to upgrade, but it is not a burn ability we need to handle via this process.

File: source/security/index.markdown (hunk 7 above)
Comment (frenck) [reply source/security/index.markdown:30]: Any access.

File: source/security/index.markdown (hunk 10 above)
Comment (frenck) [reply source/security/index.markdown:52]: > What counts as a third party library in this case? 

I don't know what to say, the sentence is clear on this.

> How would a reporter know/be able to deduce?

Not sure why that matters at all. We try to be clear upfront. If such an issue is still submitted we will close it with guidance of course.

> What would be expected if the report was about HA using a vulnerable third party library, and whatever the issue was being exploited was exposed through HA...

As the text explains, it should not reported with us.

> HA would update, but I'd suspect a CVE would probably not be needed, but a GHSA might.

Incorrect. We won't handle that as the text explains.

File: source/security/index.markdown (hunk 11 above)
Comment (frenck) [reply source/security/index.markdown:50]: We don't follow that process perse. We use the tooling for receiving report, which is what this page is about.

File: source/security/index.markdown (hunk 4 above)
Comment (frenck) [reply source/security/index.markdown]: Contact information is in the footer of every page. Will point to that.

File: source/security/index.markdown (hunk 3 above)
Comment (frenck) [reply source/security/index.markdown]: > Presuming it's fine for reports about any HA component to be reported in the core repo then?

As the text listed, if you found something, use that link.

File: source/security/index.markdown (hunk 8 above)
Comment (frenck) [reply source/security/index.markdown]: Adjusted. Except for the stable mention in the last sentence, as the page also provide pre-releases.

File: source/security/index.markdown [hunk 13]
```diff
@@ -1,11 +1,81 @@
...
+We will publish GitHub Security Advisories and through those, will also request CVEs, for valid vulnerabilities that meet the following criteria:
+
+- The vulnerability is in Home Assistant itself, not a third-party library.
+- The vulnerability is not already known to us.
+- The vulnerability is not already known to the public.
+- CVEs will only be requested for vulnerabilities with a severity of medium or higher.
+
+### Bounties
+
+As a open source project, Home Assistant cannot offer bounties for security vulnerabilities. However, if so desired, we of course will credit the discoverer of a vulnerability.
```
Comment (c0ffeeca7) [inline source/security/index.markdown]:
```suggestion
As an open source project, Home Assistant cannot offer bounties for security vulnerabilities. However, if so desired, we of course will credit the discoverer of a vulnerability.
```

**Review by c0ffeeca7 — DISMISSED** [review]

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**